migrations/educandu-2021-11-17-02-migrate-cdn-resoures.js

```diff
@@ -0,0 +1,106 @@
+// eslint-disable-next-line camelcase
+
+const pluginsWithCdnResources = new Set([
+  'image',
+  'video',
+  'image-tiles',
+  'audio',
+  'anavis',
+  'ear-training'
+]);
+
+const INTERNAL_URL_TYPE = 'internal';
+const processImageSection = ({content}) => {
+  const urls = content?.sourceType === INTERNAL_URL_TYPE ? [content.sourceUrl] : [];
```

Comment:
The type could be 'internal', but the url could be null (in all plugins)

Reply:
They are filtered out before getting saved.

Reply:
I see, nevermind